gensrc/thrift/FrontendService.thrift

```diff
@@ -531,6 +531,22 @@ struct TLoadTxnRollbackResult {
     1: required Status.TStatus status
 }
 
+struct TRLTaskCommitRequest {
+    1: required string taskId
+    2: required i64 signature
+    3: required i64 backendId
+    4: required TLoadTxnCommitRequest loadTxnCommitRequest
+    // kafka progress such as {"partitionIdToOffset":{"1":100,"2",200}}
+    5: required string progress
```

Comment:
you should define a struct for Kafka load progress

Reply:
Yes, there is a kafka progress, but not here.